src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0,
+      minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img
+        class ="product-card__image"
+        src="./images/imac.jpeg" alt="27&quot;
+        iMac computer."
+      >
+      <h2 class="product-card__description">
+        Моноблок APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)
+      </h2>
+      <p class="product-card__code">
+        Код товара: 195434
+      </p>
+      <div class="review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="review__text">Отзывов: 5</p>
+      </div>
+      <div class="price">
```

Comment:
```suggestion
      <div class="product-card__price price">
```
You can use a mix of classes here. To make `price` a separate BEM block and also as an element of `product-card'